neon_data_models/models/api/messagebus/__init__.py

```diff
@@ -210,21 +226,95 @@ class NeonSttResponse(BaseMessage):
-    
+
 
 class NeonLanguagesResponse(BaseMessage):
-    msg_type: Literal["neon.languages.get.response"] = "neon.languages.get.response"
+    msg_type: Literal["neon.languages.get.response"] = (
+        "neon.languages.get.response"
+    )
     data: NeonLanguagesData
 
 
-__all__ = [NeonGetTts.__name__, NeonGetStt.__name__, NeonTextInput.__name__,
-           NeonAudioInput.__name__, NeonSttResponse.__name__,
-           NeonTtsResponse.__name__, NeonGetLanguages.__name__,
-           NeonLanguagesResponse.__name__]
+class NeonGetSkillsApi(BaseMessage):
+    msg_type: Literal["neon.skill_api.get"] = "neon.skill_api.get"
+
+
+class NeonSkillApiData(BaseModel):
+    help: str = Field(description="API Method docstring")
+    request_schema: Optional[dict] = Field(
+        default=None, description="JSON request schema for the API method"
+    )
+    response_schema: Optional[dict] = Field(
+        default=None, description="JSON response schema for the API method"
+    )
+    signature: Optional[str] = Field(
+        default=None, description="Python signature of the API method"
+    )
+    msg_type: str = Field(
+        alias="type",
+        description="Message type associated with this API method",
+    )
+
+
+class NeonSkillsApiResponse(BaseMessage):
+    msg_type: Literal["neon.skill_api.get.response"] = (
+        "neon.skill_api.get.response"
+    )
+    data: Dict[str, Dict[str, NeonSkillApiData]] = Field(
+        description="Dict of skill_id to API methods"
+    )
+
+
+class SkillApiRequestData(BaseModel):
+    args: List[Any] = Field(default=[], description="Positional arguments")
+    kwargs: Dict[str, Any] = Field(default={}, description="Keyword arguments")
+
+
+class SkillApiResponseData(BaseModel):
+    result: Any = Field(
+        description="Result of the API method call",
+        default=None,
+    )
+    error: Optional[str] = Field(
+        default=None, description="Error message if the call failed"
+    )
+
+
+class NeonCallSkillApi(BaseMessage):
+    msg_type: str = Field(description="Requested API Message type")
+    data: SkillApiRequestData = Field(
+        default=SkillApiRequestData(),
+        description="Data for the API method call",
+    )
+
+
+class NeonCallSkillApiResponse(BaseMessage):
+    msg_type: str = Field(description="Requested API response Message type")
+    data: SkillApiResponseData = Field(description="API Method response data")
+
+
+__all__ = [
+    NeonGetTts.__name__,
+    NeonGetStt.__name__,
+    NeonTextInput.__name__,
+    NeonAudioInput.__name__,
+    NeonSttResponse.__name__,
+    NeonTtsResponse.__name__,
+    NeonGetLanguages.__name__,
+    NeonLanguagesResponse.__name__,
+    NeonGetSkillsApi.__name__,
+    NeonSkillsApiResponse.__name__,
+    NeonCallSkillApi.__name__,
+    NeonCallSkillApiResponse.__name__,
+    SkillApiRequestData.__name__,
+    SkillApiResponseData.__name__,
+    NeonSkillApiData.__name__,
```

Comment:
These should not be in `__all__` to match existing pattern of only including Message schemas